.github/workflows/docker-image.yml

```diff
@@ -33,5 +33,6 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
+          platforms: linux/amd64,linux/arm64/v8
```

Comment:
Can you please add support for windows amd64 as well? 
The image is not getting pulled in windows machine 

just add `windows/amd64` to the same, separated by comma